lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -184,50 +192,33 @@ def _finish_created_job(
             error=f"{exception}",
         )
 
-    def _prepare_to_run(self, params, concierge_params=None) -> PreparedJobParams:
+    def _prepare_to_run(self, params, concierge_params=None) -> JobSubmissionParameters:
         """
-        Creates a job record, grabs info about the objects,
-        checks the catalog resource requirements, and submits to condor
+        Creates a job record and creates the job submission params
         """
-        # perform sanity checks before creating job
-        self._check_ws_objects(source_objects=params.get("source_ws_objects"))
-        method = params.get("method")
-        # Normalize multiple formats into one format (csv vs json)
-        normalized_resources = self.sdkmr.get_catalog_utils().get_normalized_resources(
-            method
-        )
-        # These are for saving into job inputs. Maybe its best to pass this into condor as well?
-        extracted_resources = self.sdkmr.get_condor().extract_resources(
-            cgrr=normalized_resources
-        )  # type: CondorResources
```

Comment:
All this stuff is replaced by the job requirements resolver, which is run in the `_add_job_requirements` or `_get_job_reqs_from_concierge_params` methods (prior to creating the parent job in the case of the former)